packages/cli/src/commands/alias/rm.ts

```diff
@@ -61,7 +60,10 @@ export default async function rm(client, opts, args) {
     return 1;
   }
 
-  const alias = await findAliasByAliasOrId(output, now, aliasOrId);
+  const alias = await findAliasByAliasOrId(output, client, aliasOrId);
+
+  const { alias: aliasUrl } = alias;
```

Comment:
This is going to cause bugs because `alias` can be falsy as seen on the line below. Let's revert this renaming.

Reply:
Done ! Change made 👍